### PR TITLE
add equinix metal instance to CI

### PR DESCRIPTION
**Pull request by rootfs**

### CODE REVIEW

File: .github/workflows/equinix_metal_flow.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  Create-runner:
+    name: "Create Runner"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: metal-runner-action
+        uses: equinix-labs/metal-action-runner@v0.1.1
+        with:
+          github_token: ${{ secrets.GH_SELF_HOSTED_RUNNER_TOKEN }}
```
Comment (rootfs) [inline .github/workflows/equinix_metal_flow.yml:15]: @SeineSailor please review this

File: .github/workflows/equinix_metal_flow.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          metal_auth_token: ${{ secrets.EQUINIX_API_TOKEN }}
```
Comment (rootfs) [inline .github/workflows/equinix_metal_flow.yml:16]: @SeineSailor please review this

File: .github/workflows/equinix_metal_flow.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+name: Equinix Metal Action
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Create-runner:
```
Comment (wenboown) [inline .github/workflows/equinix_metal_flow.yml:7]: @SeineSailor test

File: .github/workflows/equinix_metal_flow.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          os: "ubuntu_20_04"
+
+  Install-Kepler:
+    name: "Install Kepler"
+    needs: Create-runner
+    runs-on: self-hosted
+    outputs:
+      runner-name: ${{ runner.name }}
+
+    steps:
```
Comment (rootfs) [inline .github/workflows/equinix_metal_flow.yml:29]: @SeineSailor test

File: .github/workflows/equinix_metal_flow.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+name: Equinix Metal Action
+
+on:
+  workflow_dispatch:
+
+jobs:
```
Comment (rootfs) [inline .github/workflows/equinix_metal_flow.yml:6]: @SeineSailor please review this

**Pull request closed without merging**